### PR TITLE
Add .circleci/config.yml

### PR DESCRIPTION
**Pull request by GodGreg**
Circle CI is used by infra and paramedics 

Travis-ci is not a good fit for us considering the number of tests we run (and we share it with the rest of blueprint). 

Configured to only run tests on commits to open PRs and to cancel previous tests if a commit is pushed before the other one finishes. 

https://app.circleci.com/insights/github/uwblueprint

If I have time in the future I can probably set up a home server to run the tests :).

### CODE REVIEW

File: backend/routes/submissions.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ const express = require("express");
 
 const router = express.Router();
 const db = require("../mongo.js");
-const ObjectId = require("mongodb").ObjectID;
+//const ObjectId = require("mongodb").ObjectID;
```
Comment (ahmedhamodi) [inline backend/routes/submissions.js:5]: Intentional?

**Review by ahmedhamodi — APPROVED** [review]
Small comment, I don't know if that commented out line was intentional? Rest LGTM

File: backend/routes/submissions.js (hunk 1 above)
Comment (GodGreg) [reply backend/routes/submissions.js:5]: Yes. Linter error but will be used later

Comment (ahmedhamodi) [reply backend/routes/submissions.js:5]: Cool, sounds good to me.

**Pull request merged**